frontend/src/v5/validation/drawingSchemes/drawingSchemes.ts

```diff
@@ -15,32 +15,100 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 import * as Yup from 'yup';
-import { alphaNumericHyphens, desc, name } from '../containerAndFederationSchemes/validators';
+import { revisionDesc } from '../containerAndFederationSchemes/validators';
+import { desc, name, alphaNumericHyphens, uploadFile, trimmedString } from '../shared/validators';
+import { revisionName } from './validators';
 import { formatMessage } from '@/v5/services/intl';
+import { selectRevisions } from '@/v5/store/drawings/drawingRevisions/drawingRevisions.selectors';
+import { getState } from '@/v4/modules/store';
+
+const drawingNumber = Yup.string().matches(alphaNumericHyphens,
```

Comment:
This doesn't include a character limit

Reply:
We don't yet have the backend schema (https://github.com/3drepo/3drepo.io/issues/4779), so I am not sure what the max length should be. I can add one, but we then need to remember to adjust it if not correct